webpack/private/webpack_bundle.bzl

```diff
@@ -346,6 +357,7 @@ def webpack_create_configs(name, entry_point, entry_points, webpack_config, chdi
 
 def webpack_bundle(
         name,
+        node_modules,
```

Comment:
Would having a default value here be confusing or risky?

Reply:
I think for the BREAKING CHANGE the default value would create a worst error message on upgrade. We can make a break no a minor release since we are pre-1.0 but users will still not expect it when they upgrade unless they read the release notes. An error message of `required attribute node_modules is missing` is better than `//:node_modules/webpack` target not valid if their webpack is linked somewhere else or not linked at all.

Reply:
For 1.0 we could default to `//:node_modules`. It would be nice if the error message was nicer tho if the user has there node_modules configured elsewhere.

Reply:
how about we default it to the rules_webpack installed version? and remove that option in 2.x?

Reply:
This PR removes the rules_webpack installed version. Reasoning is in the PR description. We're not at 1.0 yet so we can make breaking changes in minor releases.